bootstrap/bootstrap.go

```diff
@@ -243,3 +256,7 @@ func dumpAsJSON(data interface{}) string {
 	}
 	return string(b)
 }
+
+func GetMDPath(i int) string {
```

Comment:
don't use cryptic names for public names (decypher MD here and in other places, please). also, add godoc.